### PR TITLE
Edit/update user feature

### PR DESCRIPTION
**Pull request by mehgellan**
Add edit route and edit method to user controller
Create edit.html.erb with form for first name, last name, city, state edits
Add update route and update method to user controller with successful redirect to user profile page
Test on LH works and in db

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -8,14 +8,26 @@ def new
...
+    @user = User.create(user_params) # TODO: add error handling
     redirect_to @user
   end
 
   def show
     @user = User.find_by_id(params[:id])
   end
 
+  def edit
+    user_id = params[:id]
```
Comment (nathanallen) [inline app/controllers/users_controller.rb]: Why aren't we using `user_params` here?

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -8,14 +8,26 @@ def new
...
+    @user = User.find_by_id(user_id)
```
Comment (nathanallen) [inline app/controllers/users_controller.rb:21]: Why aren't we using `user_params` here?

Comment (mehgellan) [inline app/controllers/users_controller.rb:21]: Something like :
@user = User.find_by_id(user_params.id) ?

Comment (nathanallen) [inline app/controllers/users_controller.rb:21]: On second thought, this is okay!

**Pull request merged**